data/accountManager.go

```diff
@@ -35,10 +35,12 @@ import (
 type AccountManager struct {
 	mu deadlock.Mutex
 
+	// syncronized by mu
 	partKeys map[account.ParticipationKeyIdentity]account.PersistedParticipation
 
 	// Map to keep track of accounts for which we've sent
 	// AccountRegistered telemetry events
+	// syncronized by mu
 	registeredAccounts map[string]bool
 
 	registry account.ParticipationRegistry
```

Comment:
is registry also synchronized by mu?

Reply:
but I also see calls to other registry methods that don't use mu

Reply:
I know that before Keys() used to range over manager.partKeys but now it's been updated to only use the registry, does it still need a lock? participationDB.GetAll() has its own RW mutex.

Reply:
just did.